vendor/assets/stylesheets/dvl/components/sidebar_data.scss

```diff
@@ -61,10 +61,16 @@
 
 .sidebar_data_text {
   margin-left: $lineHeight;
-  line-height: $lineHeight;
   @media screen and (min-width: $lapWidth) {
     margin-left: $rhythm * 8;
   }
+  > div {
+    margin-bottom: 0.25rem;
```

Comment:
Why not apply this to `.sidebar_data_details` directly?

Also, you should use fractions of `$rhythm` to vertically center an element within its container.